bin/analysis_submission.py

```diff
@@ -32,12 +31,13 @@ def get_args():
     parser.add_argument('-au', '--analysis_username', help='Valid Webin submission account ID (e.g. Webin-XXXXX) used to carry out the submission', type=str, required=True)
     parser.add_argument('-ap', '--analysis_password', help='Password for Webin submission account', type=str, required=True)
     parser.add_argument('-o', '--output_location', help='A parent directory to pull configuration file and store outputs.', type=str, required=False)
-    parser.add_argument('-t', '--test', help='Specify whether to use ENA test server for submission. Options are true/T/t or false/F/f', choices=['true', 'T', 't', 'false', 'F', 'f'], required=True)
+    parser.add_argument('-as', '--asynchronous', help='Specify usage of the asynchronous Webin API for submissions. Options are true/t or false/f. Default: false/f', type=str.lower, choices=['true', 't', 'false', 'f'], required=False)
+    parser.add_argument('-t', '--test', help='Specify whether to use ENA test server for submission. Options are true/t or false/f', type=str.lower, choices=['true', 't', 'false', 'f'], required=True)
     args = parser.parse_args()
 
-    if args.test in ['true', 'T', 't']:
+    if args.test in ['true', 't']:
         args.test = True
-    elif args.test in ['false', 'F', 'f']:
+    elif args.test in ['false', 'f']:
         args.test = False
```

Comment:
There is a more elegant way to handle flags which set a boolean value to true. See here for an example of the `action='store_true'` feature in `argparse`

Reply:
Ah cool, if there's a good reason to have it this way then that's grand!